internal/gengapic/gengapic.go

```diff
@@ -257,6 +268,52 @@ func (g *generator) genAndCommitHelpers(scopes []string) error {
 	}
 	p("  }")
 	p("}")
+	p("")
+
```

Comment:
Can these HTTP and gRPC-specific funcs be moved from `genAndCommitHelpers` to somewhere where they will only be generated for clients with HTTP or gRPC methods? (`restClientInit` or similar?)

Reply:
Sure. Followup question. Are you interesting in gating these so they only generate if the transport is enabled or are you saying generate this code in the client file instead of the helper file?

Reply:
I was just thinking about gating these so they only generate if the transport is enabled .